src/main.ts

```diff
@@ -68,7 +68,7 @@ export const LunchMoneyKrakenConnection: LunchMoneyCryptoConnection<
         // Stripping the Z or X of the front works most of the time but there are some exceptions
         if (key === 'XBT' || key === 'XXBT') {
           cleaned = 'BTC';
-        } else if (key === 'XDG') {
+        } else if (key === 'XDG' || key == 'XXDG') {
           cleaned = 'DOGE';
         } else {
           cleaned = key.substr(1);
```

Comment:
a bit worried about this catch-all– what about the legitimate coins that start with X and Z? is it better to just do this as known discrepancies are brought to our attention?

Reply:
This is a good catch. This was the original logic in the connector. I had been trying to minimize modifying the existing code as much as possible, but based on your comment, it makes sense to completely redo it.